service/templates/base-auth.html

```diff
@@ -51,8 +51,16 @@
     form.addEventListener('submit', (event) => {
       /* To show user the submission is in progress */
       if (submitButtons.length === 1) {
-        submitButtons[0].setAttribute('data-state', 'busy');
-        submitButtons[0].append(...loadingContent.childNodes);
+        const submitButton = submitButtons[0];
+
+        submitButton.setAttribute('data-state', 'busy');
+        submitButton.append(...loadingContent.childNodes);
```

Comment:
**Why?** To not repeat `submitButtons[0]`, which I reference (in total) three times.